src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -68,6 +68,32 @@
  * <pre>
  * &lt;module name=&quot;EqualsAvoidNull&quot;/&gt;
  * </pre>
+ * <p>
+ * Example when ignoreEqualsIgnoreCase is set to false:
+ * </p>
+ * <pre>
+ * String nullString = null;
+ * nullString.equals("My_Sweet_String"); // violation
```

Comment:
`My_Sweet_String` please rename here and in other examples to something like "someString"